### ISSUE TITLE
french translation

### ISSUE THREAD

**Issue by tkhadir** [iluwatar/java-design-patterns#1620]
hello i think this project is indeed of french translation, do you agree with me ? @iluwatar @xiaod-dev

@all-contributors can you please add me for translation ?

**Comment by iluwatar** [comment]
@tkhadir yes, we don't have French translation at the moment. Would you like to start working on it?

**Comment by tkhadir** [comment]
Hello @iluwatar :) i would, 

great day it will be i hope my first open source contribution :)

**Comment by iluwatar** [comment]
Good to have you onboard @tkhadir, looking forward to your first pull request
